src/Nethermind/Nethermind.Evm.Test/BytecodeBuilderExtensionsTests.cs

```diff
@@ -177,6 +179,11 @@ public static IEnumerable<TestCase> opcodes_with_1_arg()
 
             foreach (Instruction opcode in address_opcodes)
             {
+                if (_releaseSpec is Shanghai && opcode is Instruction.JUMPSUB)
```

Comment:
hmm if EOF will be removed from shanghai we will have to change this code. What about IsEip4750Enabled?